backend/user_management/minio/routers/upload.py

```
@@ -0,0 +1,41 @@
+from user_management.database import get_db
+from user_management.models import Product
+from user_management.minio.utils import upload_to_minio
+from user_management.minio.schemas import ProductPhotoUploadResponse, ProductVideoUploadResponse
+
+router = APIRouter()
+
+@router.post("/upload/photos/{product_id}", response_model=ProductPhotoUploadResponse)
+async def upload_photos(
+    product_id: int, 
+    files: List[UploadFile] = File(...), 
+    db: Session = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)